vivo-project/Vitro
# [ i18n sprint] remove terms of use controller

[Companion Vitro-languages PR](https://github.com/vivo-project/Vitro-languages/pull/62)
[Companion VIVO PR](https://github.com/vivo-project/VIVO/pull/3787)

# What does this pull request do?
Removes terms of use controller, adds terms of use placeholder in companion PR

# How should this be tested?

Try in all available locales: open /termsOfUse page, verify that placeholder is present

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
